sql/mob_groups.sql

```diff
@@ -12299,6 +12299,14 @@ INSERT INTO `mob_groups` VALUES (14521,5722,288,0,128,0,0,0,105,105,0);
 INSERT INTO `mob_groups` VALUES (14522,5723,288,0,128,0,0,99999,103,104,0);
 INSERT INTO `mob_groups` VALUES (14523,5724,288,0,128,0,0,0,103,104,0);
 
+-- Ankou
+INSERT INTO `mob_groups` VALUES (14524,5726,190,3600,0,4560,650,650,25,25,0);
```

Comment:
I think you have the wrong mob droplist here - 4560 is one of the Dynamis - Qufim NMs.